src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/AccompanyingCertificates/AccompanyingCertificatesViewController.swift

```diff
@@ -33,7 +33,8 @@ class AccompanyingCertificatesViewController: DynamicTableViewController, Dismis
 		super.viewDidLoad()
 
 		navigationItem.rightBarButtonItem = dismissHandlingCloseBarButton
-		
+		navigationItem.rightBarButtonItem?.accessibilityIdentifier = AccessibilityIdentifiers.HealthCertificate.Reissuance.accompanyingCertificatesCloseButton
```

Comment:
It looks to me like the `dismissHandlingCloseBarButton ` already has the accessibility identifier `AccessibilityIdentifiers.AccessibilityLabel.close` by default, I don't think you need to introduce a new one here

Reply:
Just to document our discussion: There are two close buttons, so we need to introduce another identifier here to ensure the top-most one is tapped